guides/release/pages.yml

```diff
@@ -8,6 +8,8 @@
 - title: "Getting Started"
   url: 'getting-started'
   pages:
+    - title: "How To Use The Guides"
+      url: "intro"
```

Comment:
this should probably be renamed to index and what is currently index should be renamed to installing 😂

Reply:
the landing page is just guides.emberjs.com (equiv to guides.emberjs.com/index.html). 

how to use the guides/intro page (I didn't know what to name it) is a new page that I added, and moved the content to this location.